pending_changelog/move_netaddress_to_socketaddress.txt

```diff
@@ -0,0 +1 @@
+* The `NetAddress` has been moved to `SocketAddress`. The fieds `IPv4` and `IPv6` are also rename to `TcpIpV4` and `TcpIpV6` (#2358).
```

Comment:
PR number is off

Reply:
#2348 is the issue.  I thought it was much clearer the reason we doing this, and also linked to the PR if the user wants to see the code diff.

Reply:
Got it, I will make it right in future PR